src/mono/wasm/wasm.proj

```diff
@@ -237,6 +240,10 @@
                       Lines="@(_EmccLinkFlags)"
                       WriteOnlyWhenDifferent="true"
                       Overwrite="true" />
+    <WriteLinesToFile File="$(_EmccLinkUndefinedSymbolsFile)"
```

Comment:
Is this needed for user projects too? I don't see the file being used in `WasmApp*targets`. If it is needed only for the building the default `dotnet.wasm` then we should not add it to the runtime pack.

Reply:
No, it's not used in user projects. Are you sure it gets into the runtime pack? I thougth that runtime pack contains these files 
(and `WasmSrcFiles` doesn't contain `_EmccLinkUndefinedSymbolsFile`)
https://github.com/dotnet/runtime/blob/4c70dbb5efac51ef5b609bb19cf42e2e413f1a0b/src/mono/wasm/wasm.proj#L345-L377

What is the difference between `$(NativeBinDir)` and `$(MicrosoftNetCoreAppRuntimePackNativeDir)`?

Reply:
We seem to be using that as a staging area. I would suggest putting in `$(WasmObjDir)`, so there is no chance of accidentally copying this to the runtime pack.

Reply:
I see. I'll move it the `$(WasmObjDir)`.